src/main/resources/default/assets/common/scripts/slider.js.pasta

```diff
@@ -159,32 +153,36 @@
             _slides[currentIndex].style.opacity = '1';
         };
         const activateViewBySliding = function (direction) {
-            let containerLeft = 0, containerLeftAfterAnimation = 0, previousLeft = 0, currentLeft = 0;
+            const previousFrom = 0;//.5 * (_slider.offsetWidth - _slides[previousIndex].scrollWidth);
+            const currentTo = 0;//.5 * (_slider.offsetWidth - _slides[currentIndex].scrollWidth);
```

Comment:
Ist das Absicht? Sieht irgendwie komisch aus mit dem // mitten in der Zeile

Reply:
Ne, sorry, Debugging-Überbleibsel. Danke.

Reply:
bei JS könnts halt auch glatt ein feature sein :-P